test/tools/nl_files/subset_data_YELL

```diff
@@ -1 +1 @@
-point --lon 249.45804 --lat 44.95597 --site YELL --crop --dompft 1 --outdir EXEDIR/
+--verbose point --lon 250.45804 --lat 44.95597 --site YELL --dompft 1 --crop --create-domain --create-datm --datm-syr 2000 --create-surface --datm-eyr 2000 --outdir EXEDIR/YELL_user-mod_and_data --user-mods-dir EXEDIR/YELL_user-mod_and_data
```

Comment:
I don't really understand what this test is doing or if it's supposed to be realistic?  Are we setting the crop flag here just so the NEON cases all use the 78 pft surface dataset?  And why create the domain file?  If it's just for testing, maybe the site can be something other can a NEON site?  e.g. VERBOSE

Reply:
This is just a basic test under the tools testing framework. I just made it the same as how YELL is done currently for NEON. We could make it a different site, and remove the doman file creation, but as long as we still support the MCT option we might as well leave it in. One advantage to making it a NEON site is that you could compare it to the latest YELL site. But, since it's just a test it doesn't really have to be setup correctly and isn't necessarily going to be updated like the neon wrapper script. And if you did compare it wouldn't be identical for all fields since it didn't also have the **modify_singlept_site_neon.py** site run on it.

So if you think we should do something besides NEON, I suggest we add another test for say the US-UMB tower site. That would test both the NEON case and a more general case.

Does that help?

Reply:
- [ ] Erik will add a tools test for US-UMB
- [ ] Erik will check/change the year 2000

Reply:
I think I've addressed this. @negin513 and I will go over what I did, and I'll have her run the full set of tools tests to make sure she thinks we are covering everything.

Reply:
@negin513 and I met to talk about this. I added a few tests and went over with her how they work. I assigned her to add a regional test case, which she will do, as well as run the tools testing.